src/lib/section.svelte

```diff
@@ -1,10 +1,11 @@
 <script lang="ts">
   let clazz: string = "";
+  export let id;
```

Comment:
I can't just pass `id` into your `<Section>` component and keep it type safe. Consider this is a id pass thru.